{"url": "https://towardsdatascience.com/importing-pandas-dataframe-to-database-in-python-7212de1028c9", "time": 1683018943.065909, "path": "towardsdatascience.com/importing-pandas-dataframe-to-database-in-python-7212de1028c9/", "scrape_status": {"code": 4, "message": "invalid response status code", "resp_status_code": 429}}